novice/teaching/01-general.md

```diff
@@ -100,7 +100,7 @@ see [this wiki page](https://github.com/swcarpentry/bc/wiki/Configuration-Proble
 *   For workshops that extend over more than two days (e.g. four afternoons spread over two weeks),
     it's a good idea to email the learners at the end of each day with a summary of what was taught
     (with links to the relevant online notes).
-    Not only does this allow absent learners to catch up before the next session,
+    This allows absent learners to catch up before the next session,
```

Comment:
The original is correct: "not only x, y" is an English idiom meaning "x is true, but y is also true and probably more important".